assets/js/components/Work/Tabs/Administrative.jsx

```diff
@@ -1,75 +1,99 @@
+                  <option value="">Select dropdown</option>
+                  {visibilityOptions.map(option => (
+                    <option key={option} value={option}>
+                      {option}
+                    </option>
+                  ))}
+                </select>
+              </div>
+            ) : (
+              <p
+                className={`tag ${
+                  work.visibility === "RESTRICTED" ? "is-danger" : "is-success"
+                }`}
+              >
+                {work.visibility}
+              </p>
+            )}
+          </div>
         </div>
-        <hr />
-        <p className="notification is-warning">
-          TODO: Need to show/hide Edit UI based on editing state
-        </p>
-        <h3 className="small-title">[Edit Mode]</h3>
-        <div className="select">
-          <select>
-            <option>Select dropdown</option>
-            <option>With options</option>
-          </select>
-        </div>
-        <h3 className="small-title">Collection</h3>
-        <div className="select">
-          <select>
-            <option>Select dropdown</option>
-            <option>With options</option>
-          </select>
+
+        <div className="field">
+          <label className="label">Collection</label>
+          <div className="control">
+            {isEditing ? (
+              <div className="select">
+                <select>
+                  <option>Select dropdown</option>
+                  <option>With options</option>
+                </select>
+              </div>
+            ) : (
+              <p>
+                {work.collection
+                  ? work.collection.name
+                  : "Not part of a collection"}
+              </p>
+            )}
+          </div>
         </div>
-        <h3 className="small-title">Themes</h3>
-        <div className="select">
-          <select>
-            <option>Select dropdown</option>
-            <option>With options</option>
-          </select>
+
+        <div className="field">
+          <label className="label">Themes</label>
+          <div className="control">
+            {isEditing ? (
+              <div className="select">
+                <select>
+                  <option>Select dropdown</option>
+                  <option>With options</option>
+                </select>
+              </div>
+            ) : (
+              <p>Themes go here</p>
+            )}
+          </div>
         </div>
-        <h3 className="small-title">Preservation Level</h3>
-        <div className="select">
-          <select>
-            <option>Select dropdown</option>
-            <option>With options</option>
-          </select>
+
+        <div className="field">
+          <label className="label">Preservation Level</label>
+          <div className="control">
+            {isEditing ? (
+              <div className="select">
+                <select>
+                  <option>Select dropdown</option>
```

Comment:
These are the same as Donut: 1, 2, 3

https://github.com/nulib/donut/blob/master/config/authorities/preservation_level.yml